xml/System.Threading/CancellationToken.xml

```diff
@@ -970,30 +970,31 @@ is captured along with the delegate and is used when executing it. The current <
       <Docs>
         <summary>Throws a <see cref="T:System.OperationCanceledException" /> if this token has had cancellation requested.</summary>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This method provides functionality equivalent to:  
-  
- C#  
-  
-```csharp  
-if (token.IsCancellationRequested)   
-    throw new OperationCanceledException(token);  
-```  
-  
-```vb  
-If token.IsCancellationRequested Then  
-    Throw New OperationCanceledException(token)  
-End If  
-```  
-  
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ This method provides functionality equivalent to:
+
+ C#
+
+```csharp
+if (token.IsCancellationRequested)
+    throw new OperationCanceledException(token);
+```
+
+```vb
+If token.IsCancellationRequested Then
+    Throw New OperationCanceledException(token)
+End If
+```
+
  ]]></format>
         </remarks>
         <exception cref="T:System.OperationCanceledException">The token has had cancellation requested.</exception>
         <related type="Article" href="/dotnet/standard/threading/cancellation-in-managed-threads">Cancellation</related>
         <related type="Article" href="/dotnet/standard/parallel-programming/task-cancellation">Task Cancellation</related>
         <related type="Article" href="/dotnet/standard/parallel-programming/how-to-cancel-a-task-and-its-children">How to: Cancel a Task and Its Children</related>
+        <exception cref="T:System.ObjectDisposedException">The associated <see cref="T:System.Threading.CancellationTokenSource"/> has been disposed.</exception>
```

Comment:
It doesn't throw this exception and [i removed it recently](https://github.com/dotnet/dotnet-api-docs/pull/7555), apparently i need update the doc in the runtime repo

Reply:
Thank you for letting me know. I'll revert the changes in this file.